packages/debs/SPECS/debian/rules

```diff
@@ -89,25 +89,25 @@ override_dh_install:
 override_dh_fixperms:
 	dh_fixperms
 	# Folders
-	chown -R wazuh:wazuh $(TARGET_DIR)$(INSTALLATION_DIR)run/wazuh-server
+	chown -R wazuh-server:wazuh-server $(TARGET_DIR)$(INSTALLATION_DIR)run/wazuh-server
```

Comment:
We could declare two variables `WAZUH_USER` and `WAZUH_GROUP` to avoid repeating the name on every line